proc_Hahn_echoph.py

```diff
@@ -64,12 +40,27 @@
         logger.info(strm("there is only one dimension left -- standard 1D zeroth order phasing"))
         ph0 = ph0/abs(ph0)
     s /= ph0
-    s_sliced = s['t2':(0,None)].C
-    s_sliced['t2',0] *= 0.5
-    fl.next('time domain')
-    fl.plot(s_sliced)
-    s_sliced.ft('t2')
+    #}}}
+    #{{{visualizes the data after hermitian function test and phasing 
+    fl.next('frequency domain -- after hermitian function test and phasing')
+    s.ft('t2', pad=512) # power of 2 FT
+    s.convolve('t2',10) # so that resolution of plot isn't higher than that of screen
+    fl.image(s)
+    #}}}
+    #{{{select coherence and select t2 axis range
+    s = s['ph1',1]['ph2',0]
+    s.ift('t2')
+    s = s['t2':(0,None)]
+    s['t2',0] *= 0.5
+    s.ft('t2')
+    #}}}
+    #{{{visualize final processed data
+    fl.next('processed data')
+    fl.plot(s_uncorrected['ph2',-2]['ph1',1],
+            label='without time-axis correction',c='k')
+    fl.plot(s,label='with time-axis correction',c='r')
     fl.next('Spectrum FT')
-    fl.plot(s_sliced.real, alpha=0.5, label='real - %s'%label_str)
-    fl.plot(s_sliced.imag, alpha=0.5, label='imag - %s'%label_str)
+    fl.plot(s.real, alpha=0.5, label='real - %s'%label_str)
+    fl.plot(s.imag, alpha=0.5, label='imag - %s'%label_str)
+    #}}}
```

Comment:
here, you had oddly removed things that were in the code and done other things -- so I tried to make a mishmash that does both?

Reply:
So again I'm putting this on the fact I forgot to convert back to the newest version of the proc_Hahn_echoph after going to an archaic version. It works now with the updated stuff we had prior to me changing it to the old code for the slack convo